Functions09January/src/Max/Max.java

```diff
@@ -0,0 +1,53 @@
+
+/**
+ * 1. Calculate Max
+ * - Create a method getMax() with two integer (int) parameters, that returns maximal of the two numbers.
+ * - Write a program that reads three numbers from the console and prints the biggest of them. Use the getMax()
+ *   method you just created.
+ *
+ * @author Maariana Valentina Mitroi
+ */
+
+
+package Max;
+
+import java.util.Scanner;
+
+public class Max {
+
+    private static Scanner scanner = new Scanner(System.in);
+
+    public static void main(String[] args) {
+        exerciseOne();
+    }
+
+    /**
+     * method to get the max number out of two
+     *
+     * @param a first number to compare
+     * @param b second number to compare
+     * @return the biggest number
+     */
+    private static int getMax(int a, int b) {
```

Comment:
nice and simple implementation, good javadoc. I would avoid single letter variable, but naming them int first and int second wouldn't have made much difference here. so it's ok.